<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>예약시간</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>좌석선택</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="photo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>완료</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="seat1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777214</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="seat2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seat3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seat4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seat5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seat6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>128</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="unable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>예약중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="able">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>예약가능</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="hour1">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>1시간</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="hour4">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4시간</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="hour2">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2시간</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="hour3">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3시간</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="hour5">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5시간</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="hour6">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6시간</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
